worker/utils/utils.js

```diff
@@ -7,6 +7,15 @@ const yaml = require('js-yaml');
 const { promisify } = require('util');
 const exec = promisify(require('child_process').exec);
 const mongo = require('./mongo');
+const util= require('util');
+var crypto = require('crypto');
+
+try {
+  crypto = require('crypto');
+} catch (err) {
+  console.log(err)
```

Comment:
Would it be better to allow the tests to fail and prevent deployment that way?

Reply:
I don't believe we need this test because crypto is included in the build of nodejs we're using. I forgot to delete this code in my original pull request